tools/src/main/java/org/apache/kafka/trogdor/workload/ProduceBenchWorker.java

```diff
@@ -217,6 +216,7 @@ public Void call() throws Exception {
                 statusUpdaterFuture.cancel(false);
                 new StatusUpdater(histogram).run();
                 long curTimeMs = Time.SYSTEM.milliseconds();
+                log.info("Produced {}", payloadGenerator);
```

Comment:
Hmm, does payloadGenerator have a useful toString method?  It looks like not?  So maybe this log message is not helpful

Reply:
Yes, I implemented toString(). It outputs key size, value size, and valueDivergenceRatio